packages/arb-bridge-peripherals/contracts/tokenbridge/ethereum/gateway/L1GatewayRouter.sol

```diff
@@ -241,6 +241,32 @@ contract L1GatewayRouter is WhitelistConsumer, L1ArbitrumMessenger, GatewayRoute
         return super.outboundTransfer(_token, _to, _amount, _maxGas, _gasPriceBid, _data);
     }
 
+    function outboundTransferCustomRefund(
```

Comment:
seems to have a lot of duplicate logic with `outboundTransfer`, any reason we can't reuse the logic?

Reply:
Ah just saw your comment above

Reply:
would be good if we could a comment in the code explaining this

Reply:
added some comment to [GatewayRouter.sol](https://github.com/OffchainLabs/arbitrum/commit/0f1c0b3637bb8d4dc4de4adf9742060459258fe7#diff-7a73936444469339b89260b9eea33c240dbc90164fc1b18f1acb3bf5c9f2c33b) in commit 0f1c0b3637bb8d4dc4de4adf9742060459258fe7